IntelliTect.TestTools.Selenate/IntelliTect.TestTools.SelenateExtensions/WebElementExtensions.cs

```diff
@@ -45,9 +46,9 @@ public static void ScrollIntoView(this IWebElement element, IWebDriver driver, i
         /// then tabs out of the field or throws after a certain amount of time
         /// </summary>
         /// <param name="value">Value to send to the element</param>
-        public static async Task FillInWithAndTabWhenReady(this IWebElement element, string value)
+        public static /*async*/ void FillInWithAndTabWhenReady(this IWebElement element, string value)
```

Comment:
I would remove the commented `async`

Reply:
I'm going to revert these changes entirely and leave the project as-is, in preparation for deprecating it. In working with Kyle at a client, we've stumbled on a few key things that make it not make sense to have the extensions in a separate project (at least not these ones.) In fact, a couple of these extensions may even get moved into Browser as functions that you pass the element into.

TL;DR, reverting all changes in this class.